src/PureCloud.Client/Apis/TelephonyProvidersEdgeApi.cs

```diff
@@ -269,25 +281,1394 @@ public async Task<ExtensionPool> CreateExtensionPoolAsync(ExtensionPool body, Ca
+    public async Task<bool> DeleteExtensionPoolAsync(string extensionPoolId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(extensionPoolId);
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var response = await client.DeleteAsync($"/api/v2/telephony/providers/edges/extensionpools/{Uri.EscapeDataString(extensionPoolId)}", cancellationToken);
+        response.EnsureSuccessStatusCode();
+
+        return true;
+    }
+
+    /// <inheritdoc />
+
+    public async Task<EdgeEntityListing> GetEdgesAsync(int? pageSize = null, int? pageNumber = null, string name = null, string siteId = null, string edgeGroupId = null, string sortBy = null, bool? managed = null, CancellationToken cancellationToken = default)
+    {
+        var parameters = new NameValueCollection();
+
+        if (pageSize.HasValue)
+        {
+            parameters.Add("pageSize", pageSize.Value.ToString());
+        }
+
+        if (pageNumber.HasValue)
+        {
+            parameters.Add("pageNumber", pageNumber.Value.ToString());
+        }
+
+        if (!string.IsNullOrEmpty(name))
+        {
+            parameters.Add("name", name);
+        }
+
+        if (!string.IsNullOrEmpty(siteId))
+        {
+            parameters.Add("site.id", siteId);
+        }
+
+        if (!string.IsNullOrEmpty(edgeGroupId))
+        {
+            parameters.Add("edgeGroup.id", edgeGroupId);
+        }
+
+        if (!string.IsNullOrEmpty(sortBy))
+        {
+            parameters.Add("sortBy", sortBy);
+        }
+
+        if (managed.HasValue)
+        {
+            parameters.Add("managed", managed.Value.ToString().ToLowerInvariant());
+        }
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var uri = parameters.ToQueryString("/api/v2/telephony/providers/edges");
+        var response = await client.GetAsync(uri, cancellationToken);
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<EdgeEntityListing>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+
+    public async Task<EdgeNetworkDiagnosticResponse> GetEdgeDiagnosticNslookupAsync(string edgeId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(edgeId);
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var response = await client.GetAsync($"/api/v2/telephony/providers/edges/{Uri.EscapeDataString(edgeId)}/diagnostic/nslookup", cancellationToken);
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<EdgeNetworkDiagnosticResponse>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+
```

Comment:
Don't add new line between the comment and the method name. Apply this everywhere